src/main/java/org/gridsuite/modification/server/dto/ModificationInfos.java

```diff
@@ -100,12 +100,17 @@ public class ModificationInfos {
     @Schema(description = "Message values")
     private String messageValues;
 
+    @Schema(description = "Modification enabled")
+    @NonNull
+    private Boolean active = true;
```

Comment:
why not  @Builder.Default ? 
it would avoid setting it everywhere in tests ?

Reply:
It's something I was doing at first
But I did not like it at first because for every "toModificationInfos" methods, if you forget to call ".active(getActive)", it will be set to "false" by default

But after thinking about it, unit tests should prevent this problem
I'll make some tests, because I agree it would make everything easier

Reply:
It's way better this way, 113 modified files -> 53 files modified
fixed